llm/context/NIP46_IMPLEMENTATION.md

```
@@ -247,16 +254,18 @@ A request is auto-approved when:
 
 ### Policy from Connect URI
 
+The server prefers an explicit `invite.policy` (from the decoded connect string). If none is provided, it parses the `perms` query string.
+
 The `nostrconnect://` URI can include a `perms` parameter:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)